readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: automattic, woothemes, allendav, kellychoffman, jkudish, jeffstiel
 Tags: shipping, stamps, usps, woocommerce, taxes, payment, stripe
 Requires at least: 4.6
 Requires PHP: 5.3
```

Comment:
Woo core requires at least 5.2, and requires PHP 7. https://github.com/woocommerce/woocommerce/blob/master/readme.txt#L2-L9

Let me know if i need to bump these too.